modular_skyrat/code/game/objects/items/storage/bags.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/storage/bag/construction
+	name = "construction bag"
+	icon = 'modular_skyrat/icons/obj/tools.dmi'
+	icon_state = "construction_bag"
+	desc = "A bag for storing small construction components."
+	resistance_flags = FLAMMABLE
+
+/obj/item/storage/bag/construction/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_combined_w_class = 100
+	STR.max_items = 50
+	STR.max_w_class = WEIGHT_CLASS_SMALL
+	STR.insert_preposition = "in"
+	STR.set_holdable(list(
```

Comment:
Fairly sure this should be can_hold = list(), I did a storage refactor on tg last year which should of got rid of all these set_holdables.